orchestration/Dockerfile

```diff
@@ -9,7 +9,8 @@ ENV PYTHONFAULTHANDLER=1 \
   PIP_NO_CACHE_DIR=off \
   PIP_DISABLE_PIP_VERSION_CHECK=on \
   PIP_DEFAULT_TIMEOUT=100 \
-  POETRY_VERSION=1.1.8
+  POETRY_VERSION=1.1.8 \
+  SENTRY_DSN=https://922d76ad2dceb6cd328436ae551ffb11@o54426.ingest.sentry.io/4506559533088768
```

Comment:
Does this mean the pipeline is using the same DSN for dev and prod? Will that intermix errors between the two environments or is there a way to distinguish between the two the sentry setup?

Reply:
Yes - Sam has a setup for this [whereby each env gets a DSN](https://github.com/broadinstitute/terra-helmfile/pull/3502) which I like, but seemed like overkill for a throwaway. Plus, the only time dev runs is when I am testing the pipeline and hopefully don't have to do that again?
And - I'm still unsure of the utility here as it's all just duplicating the dagit errors which are also caught in the cloud console logs explorer.
What do you think?

Reply:
Yeah, I'm not thrilled with the idea of intermixing the environments but given the time constraints and desire to replatform probably not worth revisiting at this point.

Reply:
FWIW - if it becomes an issue in this interim, I will address it and definitely will not intermix in the re-platform